app.js

```diff
@@ -0,0 +1,29 @@
+const http = require('http');
+const config = require('./utils/config.json');
+const express = require('express');
+const bodyParser = require('body-parser');
+const fs = require('fs');
+const handlebars = require('express-handlebars');
+const logger = require('./logger');
+const morgan = require('morgan');
+const methodOverride = require('method-override');
```

Comment:
what is this?

Reply:
Old stuff left from previous tryouts. Will remove